client/app/actions/message_action_creator.coffee

```diff
@@ -22,10 +20,11 @@ MessageActionCreator =
             value: message
 
 
-    displayImages: (props={})->
+    displayImages: ({messageID, displayImages=true})->
+        message = MessageStore.getByID messageID
         AppDispatcher.dispatch
-            type: ActionTypes.SETTINGS_UPDATE_REQUEST
-            value: props
+            type: ActionTypes.SETTINGS_UPDATE_RESQUEST
```

Comment:
I have fixed the typo in the `SETTINGS_UPDATE_REQUEST` name. You can safely use it here :)